README.md

```diff
@@ -321,58 +393,144 @@ Why is a specific version of a module imported?
 
 `go mod graph | grep $modname`
 
-## Why did We Build Teleport?
-
-The Teleport creators used to work together at Rackspace. We noticed that most cloud computing users struggle with setting up and configuring infrastructure security because popular tools, while flexible, are complex to understand and expensive to maintain. Additionally, most organizations use multiple infrastructure form factors such as several cloud providers, multiple cloud accounts, servers in colocation, and even smart devices. Some of those devices run on untrusted networks, behind third-party firewalls. This only magnifies complexity and increases operational overhead.
-
-We had a choice, either start a security consulting business or build a solution that's dead-easy to use and understand. A real-time representation of all of your servers in the same room as you, as if they were magically _teleported_. Thus, Teleport was born!
-
-## More Information
-
-* [Teleport Getting Started](https://goteleport.com/docs/get-started/)
-* [Teleport
-  Architecture](https://goteleport.com/docs/reference/architecture/)
-* [Reference](https://goteleport.com/docs/reference/)
-* [FAQ](https://goteleport.com/docs/faq)
-
-## Support and Contributing
-
-We offer a few different options for support. First of all, we try to provide clear and comprehensive documentation. The docs are also in GitHub, so feel free to create a PR or file an issue if you have ideas for improvements. If you still have questions after reviewing our docs, you can also:
-
-* Join [Teleport Discussions](https://github.com/gravitational/teleport/discussions) to ask questions. Our engineers are available there to help you.
-* If you want to contribute to Teleport or file a bug report/issue, you can create an issue here in GitHub.
-* If you are interested in Teleport Enterprise or more responsive support during a POC, we can also create a dedicated Slack channel for you during your POC. You can [reach out to us through our website](https://goteleport.com/pricing/) to arrange for a POC.
-
-## Is Teleport Secure and Production-Ready?
-
-Yes -- Teleport is production-ready and designed to protect and facilitate
-access to the most precious and mission-critical applications.
-
-Teleport has completed several security audits from nationally and
-internationally recognized technology security companies.
-
-We publicize some of our audit results, security philosophy and related
-information on our [trust page](https://trust.goteleport.com/).
-
-You can see the list of companies that use Teleport in production on the Teleport
-[product page](https://goteleport.com/case-study/).
-
-## Who Built Teleport?
-
-Teleport was created by [Gravitational, Inc.](https://goteleport.com). We have
-built Teleport by borrowing from our previous experiences at Rackspace. [Learn more
-about Teleport and our history](https://goteleport.com/about/).
+## Teleport Editions & Features
```

Comment:
I'm concerned that this will go out of date and drift from our docs page that already mentions this information.

Can we link to https://goteleport.com/docs/feature-matrix/ so we have a single source of truth instead?